hack/install-envtest.sh

```diff
@@ -32,6 +32,6 @@ version=$(cat ${SCRIPT_ROOT}/go.mod | grep 'k8s.io/kubernetes' | grep -v '=>' |
 
 GOPATH=$(go env GOPATH)
 TEMP_DIR=${TMPDIR-/tmp}
-go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+go install sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.17
```

Comment:
could you change to 0.15 to align with go.mod:

https://github.com/kubernetes-sigs/scheduler-plugins/blob/cd3e4fb738722a7e07b10e4be93eff7285f39f73/go.mod#L31

Reply:
Ok, I've fixed it.